.github/workflows/deploy_tre_reusable.yml

```diff
@@ -336,6 +337,7 @@ jobs:
           TF_VAR_application_admin_client_secret: "${{ secrets.APPLICATION_ADMIN_CLIENT_SECRET }}"
           TF_VAR_stateful_resources_locked:
             "${{ github.ref == 'refs/heads/main' && inputs.prRef == '' && true || false }}"
+          TF_VAR_enable_airlock_malware_scanning:  "${{ secrets.ENABLE_AIRLOCK_MALWARE_SCANNING }}"
```

Comment:
I don't have anywhere else to put this is but I believe you're missing changes in other workflows that call on this one: prbot, deploy-tre, deploy-tre-branch